docs/index.bs

```diff
@@ -1248,6 +1256,7 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
                 1. If |client|'s [=environment/execution ready flag=] is unset or |client|'s [=discarded flag=] is set, [=continue=].
                 1. If |client| is not a [=secure context=], [=continue=].
                 1. If |options|["{{ClientQueryOptions/includeUncontrolled}}"] is false, and if |client|'s [=active service worker=] is not the associated [=ServiceWorkerGlobalScope/service worker=], [=continue=].
+                1. If |options|["{{ClientQueryOptions/includeFrozen}}"] is false, |client| is {{Client/frozen}}, [=continue=].
```

Comment:
`|client|` is a `[=service worker client=]`, not a `{{Client}}`, so `{{Client/frozen}}` doesn't make sense here.

At this point you need to determine if the client is actually frozen or not. Also, be wary of race conditions here. Ensure that, if `includeFrozen` is false, no frozen clients are returned.